internal/confmapprovider/discovery/bundle/bundle_gen.tmpl

```diff
@@ -0,0 +1,33 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build bundle.d
+
+// These are the discovery config component generating statements.
+// In order to update run go generate -tags bundle.d ./...
+
+{{- range $ext := .Extensions }}
```

Comment:
Asking a simple minded probing question here. Should there be a section for windows support?

Reply:
We don't currently bundle discovery config for windows packages, but there'd be one when we do.